src/interfaces.ts

```diff
@@ -40,3 +40,13 @@ export interface GetAuthURLOptions {
   screenHint?: 'sign-up' | 'sign-in';
   returnPathname?: string;
 }
+
+export interface AuthkitMiddlewareAuth {
+  enabled: boolean;
```

Comment:
do we need this extra `enabled` key?

Reply:
Unfortunately I believe we do. Having `unauthenticatedPaths` isn't enough to determine whether a user wants to use middleware auth or not. 

You could argue that if the `authenticatedPaths` array is empty then that means that the user doesn't want to use middleware auth. However an edge case is that someone wants middleware auth on _all_ routes with no exception, in which case an empty array makes sense.

Having an explicit `enabled` boolean means there can be no confusion on what the user's intention is.

Reply:
I see, I guess it doesn't hurt to be extra explicit.